manager.go

```diff
@@ -179,12 +182,10 @@ func (m *Manager) Restart() (*os.Process, error) {
 func (m *Manager) childWait(rc io.ReadCloser) error {
 	for {
 		buf := make([]byte, 10)
-		n, err := rc.Read(buf)
+		_, err := rc.Read(buf)
```

Comment:
Convert this to an inline now there's no requirement to have the vars available later.